app/jobs/notifications/send_notification_job.rb

```diff
@@ -9,7 +9,8 @@ def perform notification, owner, recipient, content, invoice, click_action
     end
     text = I18n.t("notifications.#{content}", user_name: owner.name, invoice_name: invoice.name)
     options = {notification: {title: I18n.t("notifications.app_name"), text: text,
-      click_action: click_action}, data: {notification_id: notification.id, invoice_id: invoice.id}}
+      click_action: click_action}, data: {notification_id: notification.id,
```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: com.puppycrawl.tools.checkstyle.Style/SpaceInsideHashLiteralBraces
**Severity**: INFO
**File**: app/jobs/notifications/send_notification_job.rb L12

Space inside { missing.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*